relay-event-normalization/src/validation.rs

```diff
@@ -22,51 +37,107 @@ pub struct TransactionValidationConfig {
 
     /// Controls whether the event has been validated before, in which case disables validation.
     ///
-    /// By default, `is_validated` is disabled and transaction validation is run.
+    /// By default, `is_validated` is disabled and event validation is run.
```

Comment:
We should rename `timestamp_range` to `transaction_timestamp_range` because it does not apply to error events.